### DISCUSSION TITLE
Resize and set position depending on taskbar visible or hidden

### DISCUSSION THREAD

**Discussion by ManCaveMedia** [f1multiviewer/issue-tracker#156] [Feature requests] [answered]
On Windows 10 when you set a video size to ie. 'Half screen' it scales to half of the _full_ display size, not half of the _available_ display size (_full_ display size minus the taskbar height). When you position the video to ie. 'Bottom Left', the bottom left corner of the video is positioned at the bottom left corner of the _available_ display size, not at the bottom left of the _full_ display height. (When you have the taskbar at the bottom, 'Bottom' means 'on top of the taskbar' while positioning.)This feels inconsistent.

It would be nice to have an option in settings to use the 'available display height' or 'full display height' and mak it consistent for both options.

**Comment by jeppeolesen** [comment, marked as answer]
I believe this is a consequence of the Windows API when setting window positions and sizes.

Additionally, since this feature request was made, "Layouts" have been introduced in MultiViewer, meaning you can create pre-defined layouts and load them when opening a session, so you won't have to reposition the players every time which can be used to position your windows as intended and then save as a layout for later use.
